apps/web/src/components/changes/ChangesTableLayout.tsx

```diff
@@ -27,17 +37,17 @@ export const ChangesTable = ({ changes, loading }: { changes: Data[]; loading: b
             <Text color={colorScheme === 'dark' ? colors.B40 : colors.B70}>Message Change</Text>
           )}
           <Text rows={1} mt={5}>
-            {change}
+            {JSON.stringify(change)}
```

Comment:
Not sure about this one, I think we should just show the change type: If it's a template or message change and show the template name in this case. Stringifying the whole change object will probably not look so good.

Reply:
@ainouzgali

Reply:
Yeah this was one of my questions 😄